plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveNodePartitioningProvider.java

```diff
@@ -95,23 +88,7 @@ public ConnectorBucketNodeMap getBucketNodeMap(ConnectorTransactionHandle transa
         // However, number of partitions is not known here
```

Comment:
Should we remove this comment block?

Reply:
Comment is mostly relevant. I will rework it